apps/examples/testcase/ta_tc/media/utc/utc_media_player.cpp

```diff
@@ -54,7 +54,7 @@ TEST_F(SimpleMediaPlayerTest, getState)
 
 TEST_F(SimpleMediaPlayerTest, create)
 {
-	ASSERT_EQ(mp->create(), media::PLAYER_OK);
+	EXPECT_EQ(mp->create(), media::PLAYER_OK);
 	EXPECT_EQ(mp->getState(), media::PLAYER_STATE_IDLE);
```

Comment:
please , remove getstate() function.
When the user and our framework access that function, the state is incorrect.

Reply:
I'll change in another PR